src/messages/lang_fr-FR.json

```diff
@@ -7,16 +7,16 @@
     "lang-nl-NL": "Nederlands",
     "lang-sv-SE": "Svenska",
 
-    "navButtonTooltip": "Sélectionnez pour afficher d'autres langues",
+    "navButtonTooltip": "Sélectionner pour afficher d'autres langues",
     "lang-en-US-tooltip": "Select for English",
-    "lang-fr-FR-tooltip": "Sélectionnez pour le français",
+    "lang-fr-FR-tooltip": "Sélectionner pour le français",
     "lang-es-MX-tooltip": "Selecciona Español",
     "lang-de-DE-tooltip": "Wählen Sie für die Deutsche",
     "lang-nl-NL-tooltip": "Kies voor Nederlands",
     "lang-sv-SE-tooltip": "Välj för Svenska",
 
     "lang-en-US-tooltipAtSelect": "English is currently selected",
-    "lang-fr-FR-tooltipAtSelect": "Français est actuellement sélectionné",
+    "lang-fr-FR-tooltipAtSelect": "Le français est actuellement sélectionné",
```

Comment:
The french change for "lang-fr-FR-tooltipAtSelect" should be in other lang json files as well.